ui/src/authors/__tests__/utils.test.js

```diff
@@ -19,4 +19,66 @@ describe('utils', () => {
       expect(result).toBe(List());
     });
   });
+
+  describe('getAuthorMetaDescription', () => {
+    it('returns full meta description', () => {
+      const author = fromJS({
+        name: { native_names: ['エドウィン'] },
+        control_number: 12345,
+        project_membership: [{ name: 'CERN-LHC-CMS' }, { name: 'CERN-AMS' }],
+        positions: [
+          { institution: 'CERN', current: true },
+          { institution: 'MIT' },
+          { institution: 'DESY', current: true },
+        ],
+        arxiv_categories: ['hep-th', 'hep-ex'],
+      });
+      const expected = 'エドウィン. CERN and DESY. hep-th and hep-ex. CERN-LHC-CMS and CERN-AMS'
```

Comment:
what's the case with preferred_name and native_names? do we display both?

Reply:
nope, only native_names, the real name is in the `<title>`